app/script/auth/AuthService.js

```diff
@@ -198,13 +201,11 @@ z.auth.AuthService = class AuthService {
         },
         processData: false,
         type: 'POST',
-        url: `${this.client.create_url(`${AuthService.CONFIG.URL_LOGIN}?persist=${persist}`)}`,
+        url: `${this.client.create_url(AuthService.CONFIG.URL_LOGIN)}?persist=${(persist === true ? 'true' : 'false')}`,
```

Comment:
At least:

`${this.client.create_url(AuthService.CONFIG.URL_LOGIN)}?persist=${window.encodeURIComponent(persist.toString())}`

then